content/tutorials/geodesic-getting-started.md

```diff
@@ -0,0 +1,98 @@
+
+1. Geodesic overrides the default entrypoint with a custom script that enables starting a login shell or installing geodesic as an executable on your local machine. If you don't know much about Docker's `ENTRYPOINT` or `CMD` capabilities then [we recommend reading this article before moving forward](https://phoenixnap.com/kb/docker-cmd-vs-entrypoint).
+1. Geodesic can be thought of as just another shell that you open up on your machine like `zsh` or `bash`, but when running it inside of Docker means that it doesn't have access to your home directory or the projects you're looking to work on. To get around that, Geodesic is typically run using Docker volume bind mounts via the `--volume` / `-v` flag. You'll see this in our examples below, but if you would like to understand more about what is going on under the hood then we recommend [reading up quickly on mounting local volumes via `docker run`](https://docs.docker.com/engine/reference/commandline/run/#mount-volume--v---read-only).
+
+### Geodesic Usage Patterns
+
+Let's talk about a few of the ways that one can run Geodesic. Our toolbox has been built to satisfy many use-cases, and each result in a different pattern of invocation:
+
+1. You can **run standalone** Geodesic as a standard docker container using `docker run`. This enables you to get started quickly, avoid fiddling with configuration, or run a one off commands using some of the built in tools.
+   1. Example: `docker run -it --rm --volume $HOME:/localhost cloudposse/geodesic:latest-debian --login` opens a bash login shell (`--login` is our Docker `CMD` here) in our Geodesic container.
+   1. Example: `docker run -it --rm --volume $HOME:/localhost cloudposse/geodesic:latest-debian -c "terraform version"` executes the `terraform version` command as a one off and outputs the result.
+1. You can **install** Geodesic onto your local machine using what we call the docker-pipe-bash pattern. Similar to above, this enables a quickstart process but supports longer lived usage as it creates a callable script on your machine that enables reuse.
+   1. Example: `docker run --rm cloudposse/geodesic:latest-debian | bash -s latest-debian` installs `/usr/local/bin/geodesic` on your local machine which you can execute repeatedly via simply typing `geodesic`
+1. Lastly, you can **build your own toolbox** on top of Geodesic. Want to provide additional packages to your team on top of what geodesic provides or want to customize the name of Geodesic to the name of your organization? This is simple to do by using Geodesic as your base image in your own Dockerfile, adding your own Docker `RUN` commands or overriding environment variables, and then building a new image which you distribute to your team. This is more advanced usage and we'll cover how to do this in a future how-to article.
```

Comment:
```suggestion
1. Lastly, you can **build your own toolbox** on top of Geodesic. This is what we generally recommend to our customers. We do then when we want to provide additional packages or customization to our team while building on the foundation that geodesic provides. This is simple to do by using Geodesic as your base image in your own `Dockerfile`, adding your own Docker `RUN` commands or overriding environment variables, and then building a new image that you distribute to your team. This is more advanced usage and we'll cover how to do this in a future how-to article.
```

Reply:
@osterman updated this one a tiny bit so as to avoid using "we" from the Cloud Posse perspective since we've been speaking as if the first person is the methodology, not Cloud Posse.